js_modules/dagster-ui/packages/ui-components/src/components/TextInput.tsx

```diff
@@ -15,6 +16,7 @@ interface Props extends Omit<React.ComponentPropsWithRef<'input'>, 'onChange'> {
 export const TextInput = React.forwardRef(
   (props: Props, ref: React.ForwardedRef<HTMLInputElement>) => {
     const {
+      fill,
```

Comment:
This is an option on Button and Select and it will allow us to clean up two more custom CSS overrides in Plus -- because there's a container div on line 29 that doesn't take any external styles, it's been difficult to make a full-width input